api/v1/model/src/commonMain/kotlin/User.kt

```diff
@@ -27,7 +27,14 @@ import kotlinx.serialization.Serializable
 @Serializable
 data class User(
     val id: String,
-    val username: String
+    val username: String,
+)
+
+@Serializable
+data class CreateUser(
+    val username: String,
+    val password: String? = null,
+    val temporary: Boolean = true
```

Comment:
This and OpenAPI should document what temporary does.

Reply:
@murto I couldn't find any examples from `ktorswaggerui` where the descriptions could be given at the field level, only at body level.

Reply:
I just noticed that my answer above was stuck in pending state since yesterday...

Reply:
@mnonnenmacher I'm sorry but I lack the knowledge of how to use that. Is it a library that needs to be imported from somewhere?  The example you linked is lacking on some crucial info (at least to me), and as such, IntelliJ doesn't recognize it, so obviously the `User.kt` file is lacking an import.

Reply:
To me that's not mandatory, we do not use that anywhere else.